luda-editor/client/src/pages/sequence_edit_page/sequence/mod.rs

```diff
@@ -69,6 +69,9 @@ impl SequenceWrapped {
                     cut_id,
                     after_cut_id,
                 } => {
+                    if after_cut_id == Some(cut_id) {
+                        panic!("Cannot move cut after itself");
+                    }
```

Comment:
You didn't get what i'm saying.
You don't have to make a panic when it can be filtered.
Do filter at this line, so stop sending meaningless request to client and server.

Why did you make a panic here?

Reply:
It seems blocked by this code.
https://github.com/NamseEnt/namseent/blob/8cb4c8a37ec98a5a8bf9f80836dc1a4898afdefa/luda-editor/client/src/pages/sequence_edit_page/components/cut_list_view/mod.rs#L160-L171

Why did you need this kind of panic? for what case?

Reply:
Developer may accidentally forget to filter that case while developing client someday.
At that time, the server should still check the case so that the developer can notice the error as soon as possible.

Reply:
I don't get it. 
Just make assert than, not if-and-panic.
And this PR title and description is not about this assertion.

Why do you want to this? what's the purpose? Preventing future error?

Don't block server. just block client. It's very logically same that server code could be changed so it accidently forget to filtering this case.

Reply:
Purpose of this pr was ignoring this edge case that causing below problem.
> If after_* is indicating self, Inserting after after_* is impossible because self is already removed.
So ignore order change request if after_* is indicating self.

And purpose is changed to preventing future error with assertion after your comment below.
> Without error, nobody can catch that leak.
Makes error as early as you can, so make a service better.
No error is best, but hiding error is not the best.

Reply:
> Don't block server. just block client.

Why we should filter this case at server or client one-side?
Isn't it right to filter both side because we can make mistakes on both sides?

Reply:
Both side could be changed at the same time. Adding `assert!` doesn't solve the problem.

Hey, then why you make the client, instead of remote mirroring like xbox or steam cloud play?

We can believe client because even if user abuse client, it doesn't affect to other project and our business.
So you don't have to make a complex server which react to everything about client-side happening.
Server only save data in database, that's the only different thing about cut editing against client.
The storage layer is different between server and client so that's why we have to implement Action's handler in server too, not for security or bug-catch reason.

So, you said the purpose is preventing index problem.
Then make a assertion at client side if you need.
Do we need? actually I don't have any idea about it. Why need it?
If you really need it, make a data structure which prevent this problem in general. Then it super force programmer don't make that kind of coding error.

Reply:
Don't make program complex. We are the only people who must maintain it.
I want simple and well-maintainable code.

No 3-place assertion. I very disagree with current PR's suggestion.

Reply:
I got a good idea
How about adding a test for that case?

Reply:
Nah. nevermind.